Firestore/Example/Tests/Util/FSTIntegrationTestCase.h

```diff
@@ -90,6 +95,9 @@ extern "C" {
  */
 - (void)waitUntil:(BOOL (^)())predicate;
 
+/** Returns a completion block that fulfills a newly-created expectation with the specified name. */
+- (FSTVoidErrorBlock)completionExpectationWithName:(NSString *)name;
```

Comment:
This feels a little bit out of place. What about putting it in XCTTestCase+Await?

I would also suggest to rename this to `fulfillExpectationWithName`?

Reply:
Good call about moving to XCTTestCase+Await. I didn't know that existed.

I'm not wild about fulfillExpectationWithName since I think Obj-C dictates it should be a noun-y name indicating it returns a completionBlock. I think `completionBlockFulfillingExpectationWithDescription:` would be the absolutely precise name, but that's a bit verbose. :-)

WDYT of `completionBlockExpecting:`? Other ideas?

Reply:
How about `completionForExpectationWithName`?

Reply:
Works for me. Done.